http.go

```diff
@@ -9,13 +9,14 @@ import (
 	"strings"
 	"time"
 
+	"github.com/oauth2-proxy/oauth2-proxy/pkg/apis/options"
```

Comment:
Options aren't an "api"...

Reply:
I'd argue that they are. The options represent the configuration that a user will need to tell our program how to function. The create an external API contract with users. If we change the configuration in a breaking way, that breaks the user functionality. Therefore requires a version bump. Ie, the options here, behave exactly the same as an API.